### DISCUSSION TITLE
Unable to use php 8 attributes

### DISCUSSION THREAD

**Discussion by bradley-r-martin** [spatie/typescript-transformer#83] [Q&A]
Having some issues using the PHP 8 attributes.

Using the comment based /** @typescript */ declaration works fine however i need to be able to use the other attributes like: #[LiteralTypeScriptType("string | null")]

Ive confirmed that php 8.3.8 is running. 

Running in laravel 11. 

Any ideas why this might be ?

**Comment by rubenvanassche** [comment]
This should normally be working, could you present us a test to show what's not working?
